src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePrev = sourceString.trim().split(';');
```

Comment:
The variable name 'stylePrev' is not descriptive enough and could be improved for better readability. Consider a name that reflects the content of the variable, such as 'styleDeclarations' or 'styleEntries'.